src/plugins/data_source_management/public/components/data_source_view/data_source_view.tsx

```diff
@@ -5,22 +5,24 @@
 
 import React from 'react';
 import { i18n } from '@osd/i18n';
-import { EuiPopover, EuiButtonEmpty, EuiButtonIcon, EuiContextMenu } from '@elastic/eui';
+import { EuiButtonEmpty, EuiToolTip, EuiFlexGroup, EuiFlexItem, EuiBadge } from '@elastic/eui';
 import { SavedObjectsClientContract, ToastsStart } from 'opensearch-dashboards/public';
+import { IUiSettingsClient } from 'src/core/public';
 import { DataSourceOption } from '../data_source_menu/types';
 import { getDataSourceById } from '../utils';
-import { MenuPanelItem } from '../../types';
 
 interface DataSourceViewProps {
   fullWidth: boolean;
   selectedOption: DataSourceOption[];
-  savedObjectsClient?: SavedObjectsClientContract;
-  notifications?: ToastsStart;
+  savedObjectsClient: SavedObjectsClientContract;
```

Comment:
SavedObjctClient and notification should be required according to this doc: https://quip-amazon.com/UepCAPYFGtKC/Use-Data-Source-Menu-to-Nav-Bar-or-Data-Source-Selector-inside-Page